siliconcompiler/schema.py

```diff
@@ -1808,6 +1807,23 @@ def schema_flowgraph(cfg, step='default', index='default'):
         """
     }
 
+    cfg['flowgraph'][flow][step][index]['flowin'] = {
```

Comment:
should we mark WIP or unsupported somehow? I know we haven't done this for some other unsupported things, but I get a little worried about creating confusion by adding in placeholders to schema.

Reply:
Removed, you are right!